### PR TITLE
Add push notification config snippets

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

File: ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Your Chat Service SID from https://www.twilio.com/console/chat/services
+service_sid = 'ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+# Authenticate with Twilio
+@client = Twilio::REST::Client.new(api_key, api_secret, account_sid)
+
+# Update the service webhooks
+service = @client.chat.v2.services(service_sid)
+                 .update(notifications_added_to_channel_enabled: true,
+                         notifications_added_to_channel_template:
+          'A New message in ${CHANNEL} from ${USER}: ${MESSAGE}',
```
Comment (acamino) [inline ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb]: @hortega, indentation looks weird on this line.

File: ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# Find your Account Sid at https://www.twilio.com/console/account/settings
+account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+# Create an API Key and Secret at https://www.twilio.com/console/chat/dev-tools/api-keys
+api_key = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+api_secret = 'your_api_secret'
+# Your Chat Service SID from https://www.twilio.com/console/chat/services
+service_sid = 'ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+# Authenticate with Twilio
+@client = Twilio::REST::Client.new(api_key, api_secret, account_sid)
```
Comment (acamino) [inline ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb]: @hortega, I left a comment in another pull request, regarding the use of instance variables in the Ruby snippets. Would you mind taking a look at it, please? https://github.com/TwilioDevEd/api-snippets/pull/520#discussion_r141432264

I'd love to have your comments on this.

Comment (hortega) [reply ip-messaging/push-config/new-message-push-config/new-message-push-config.5.x.rb]: Yes, I also agree the `@` is unnecesary.

**Comment by hortega** [comment]
Thanks @acamino !

**Pull request merged**